lib/std/child_process.zig

```diff
@@ -277,6 +277,7 @@ pub const ChildProcess = struct {
             .capacity = fifo.buf.len,
             .allocator = fifo.allocator,
         };
+        fifo.deinit();
```

Comment:
Hm, is this right? We've stolen the contents of fifo on line 275, no? That is, data used to be owned by `fifo` is now owned by `result`, and its on the caller to release it?

Reply:
Indeed, this PR is wrong as written!